packages/astro/src/assets/services/sharp.ts

```diff
@@ -12,7 +12,7 @@ export interface SharpImageServiceConfig {
 	/**
 	 * The `limitInputPixels` option passed to Sharp. See https://sharp.pixelplumbing.com/api-constructor for more information
 	 */
-	limitInputPixels?: number;
+	limitInputPixels?: number | boolean;
```

Comment:
Is there any way we could somehow use the type from Sharp's directly?

Reply:
Good point. For some reason I never thought about it. Updated to use the type from Sharp.